lib/pages/completed_tasks.dart

```diff
@@ -6,90 +6,104 @@ import 'package:task_list_app/constants/constants.dart';
 import 'package:task_list_app/cubit/task_cubit.dart';
 import 'package:task_list_app/models/task.dart';
 
-class CompletedTaskScreen extends StatefulWidget {
+class CompletedTaskScreen extends StatelessWidget {
   const CompletedTaskScreen({super.key});
 
-  @override
-  State<CompletedTaskScreen> createState() => _CompletedTaskScreenState();
-}
-
-class _CompletedTaskScreenState extends State<CompletedTaskScreen> {
   @override
   Widget build(BuildContext context) {
-    context.read<TaskCubit>().readTasks();
-    return BlocBuilder<TaskCubit, TaskState>(builder: (context, state) {
-      List<Widget> completedTasks = state.tasks.map(
-        (task) {
-          return ListTile(
-            key: const Key('completed tiles'),
-            leading: IconButton(
-              icon: const Icon(Icons.check),
-              color: Colors.green,
-              onPressed: () {},
-            ),
-            title: Text(
-              task.name,
-              style: tilesText,
-            ),
-            trailing: IconButton(
-              onPressed: () => _onTapTrashIcon(task),
-              icon: const Icon(
-                Icons.delete,
-                color: Colors.redAccent,
+    context.read<TaskCubit>().readTasksByStatus(TaskStatus.completed);
+    return BlocBuilder<TaskCubit, TaskState>(
+      builder: (context, state) {
+        List<Widget> completedTasks = state.tasks.map(
+          (task) {
+            return ListTile(
+              key: const Key('completed tiles'),
```

Comment:
I would recommend changing this to be separated by underscores like the majority of the `Key`s. Spaces can be an issue in general.